astropy/table/tests/test_showtable.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+import os
+
+from ..scripts import showtable
+
+ROOT = os.path.abspath(os.path.dirname(__file__))
+ASCII_ROOT = os.path.join(ROOT, '..', '..', 'io', 'ascii', 'tests')
+FITS_ROOT = os.path.join(ROOT, '..', '..', 'io', 'fits', 'tests')
+VOTABLE_ROOT = os.path.join(ROOT, '..', '..', 'io', 'votable', 'tests')
+
+
+def test_missing_file(capsys):
+    showtable.main(['foobar.fits'])
+    out, err = capsys.readouterr()
+    assert err.startswith("ERROR: [Errno 2] No such file or directory: "
+                          "'foobar.fits'")
+
+
+def test_info(capsys):
+    showtable.main([os.path.join(FITS_ROOT, 'data/table.fits'), '--info'])
+    out, err = capsys.readouterr()
+    assert out == ('<Table length=3>{0}'
+                   ' name   dtype {0}'
+                   '------ -------{0}'
+                   'target   str20{0}'
+                   ' V_mag float32{0}'
+                   '\n').format(os.linesep)
+
+
+def test_stats(capsys):
+    showtable.main([os.path.join(FITS_ROOT, 'data/table.fits'), '--stats'])
+    out, err = capsys.readouterr()
+    assert out == ('<Table length=3>{0}'
+                   ' name    mean    std   min  max {0}'
+                   '------ ------- ------- ---- ----{0}'
+                   'target      --      --   --   --{0}'
+                   ' V_mag 12.8667 1.72111 11.1 15.2{0}').format(os.linesep)
+
+
+def test_fits(capsys):
+    showtable.main([os.path.join(FITS_ROOT, 'data/table.fits')])
+    out, err = capsys.readouterr()
+    assert out == (' target V_mag\n'
```

Comment:
I see tests are passing, but why don't these need the `os.linesep` fix?  Is this a bug in `Table.pprint` that it isn't using `os.linesep`?

Reply:
The behavior differs (`pprint` just prints each line, whereas `info` writes to stdout using `os.linesep` explicitly) but I don't know what behavior is expected for terminal output (as far as I know the convention differs for *files* ?).